### ISSUE TITLE
❓ - Third-party cookie will be blocked. Learn more in the Issues tab.

### ISSUE THREAD

**Issue by mstgnz** [ddubrava/angular8-yandex-maps#236]
### Description

Do you have any information about this error?

```html
<div class="container" style="width: 100%;" [style.height]="height">
  <ya-map [center]="center" [options]="mapOptions">
      <ya-multiroute 
        [referencePoints]="markers"
        [options]="multirouteOptions"
      ></ya-multiroute>
  </ya-map>
</div>
```

```error
Console Error: Third-party cookie will be blocked. Learn more in the Issues tab.
```

**Comment by ddubrava** [comment]
You need to open the issues tab, there are affected resources. I don't think the problem is with the library. Even if it is, the library doesn't manage cookies at all

**Comment by mstgnz** [comment]
Only when I used maps, this warning appeared and the map would not open. It was fixed when I allowed "third-party cookie" in the browser.

But it is a problem to explain this to users. either this package uses "third-party cookie" or yandex uses it itself.

I found a statement that it was deprecated
[the-end-of-3rd-party-cookies-chrome-is-deprecating](https://medium.com/@justingoldberg_2282/the-end-of-3rd-party-cookies-chrome-is-deprecating-them-in-2024-335bb565ef5f)

<img width="693" alt="Screenshot 2024-02-20 at 09 24 08" src="https://github.com/ddubrava/angular8-yandex-maps/assets/9930661/ba8f2474-ad23-4f42-ac5a-8cf4004f75e8">
